plugin.rb

```diff
@@ -23,19 +23,33 @@
       Sentry.init do |config|
         config.dsn = SiteSetting.quipper_sentry_dsn
         config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+        
+        # ignore ApplicationController::RenderEmpty
+        # ApplicationController::RenderEmpty happened 
+        # when some controller that extends ApplicationController not call `render`
+        # config.before_send = lambda do |event, hint|
+        #   message = event.dig(:logentry, :formatted)
+        #   return nil if message.include?('ApplicationController::RenderEmpty')
+        #   event
+        # end
```

Comment:
Do we still need this comment @dirham? 🤔

Reply:
Yes, until we decide to ignore or not the sentry report for this: ApplicationController::RenderEmpty